.github/workflows/check-pull-request.yml

```diff
@@ -21,13 +21,10 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  CARGO_ARGS: ${{ github.ref == 'refs/heads/develop' && '--release' || '' }}
   CARGO_TERM_COLOR: always
-  SKIP_WASM_BUILD: 1
```

Comment:
This should no-longer be global as the zombienet should build wasm - I think the default execution mode is wasm already.